tests/publiccloud/create_aistack_env.pm

```diff
@@ -162,14 +177,41 @@ sub install_aistack_chart {
     }    #pod loop
 
     assert_script_run("kubectl get all --namespace $namespace");
-    record_info("Logs for the pods which is not in running or pending state");
-    foreach my $pod (@out) {
-        my $status = script_output("kubectl get pod $pod -n $namespace -o=jsonpath='{.status.phase}'", proceed_on_failure => 1);
-        if ($status !~ /^(Running|Pending|Completed)$/) {
-            my $logs = script_output("kubectl logs $pod -n $namespace", proceed_on_failure => 1);
-            record_info("$pod is in $status state. Logs:\n$logs\n");
-        }
-    }    # pod loop
+    if (@failed_pods) {
+        record_info("Failed pods:", join("\n", map { "$_->{name}: $_->{status}" } @failed_pods));
+        exit 1;
```

Comment:
I am not sure if `exit` will do the right thing , it is more common just to use `die`